src/olympia/constants/webext_permissions.py

```diff
@@ -16,6 +16,7 @@
     u'http://*/*': ALL_URLS_PERMISSION,
     u'https://*/*': ALL_URLS_PERMISSION,
     u'*://*/*': ALL_URLS_PERMISSION,
+    # FIXME: how about https://*/path ?
```

Comment:
fwiw, I'm changing it from a list to `r'^\<all_urls\>|(\*|http|https):\/\/\*\/\*'`, which we can tweak, in the pull for #4540